### ISSUE TITLE
inconsistent results using fuzzywuzzy library

### ISSUE THREAD

**Issue by kd10041** [seatgeek/thefuzz#75]
So, I have used this library for my workflow to match a bunch of pairs of strings. But this whenever I run the script the output results varies ! How can this be possible shouldn't the levenshtein distance between two strings is always constant (matching the order of the strings) ?

**Comment by maxbachmann** [comment]
Yes the similarities should stay the same. Can you provide an example where this does occur?

**Comment by kd10041** [comment]
Currently I don't have any concrete examples but I am using this library to match a pairs of a bunch of datas ( ~800 ) So when I run the script with same datafiles and script on my pc and on my colleague's pc the output varies! 
for context: I am trying to match a smaller string ( all in english ) to a larger string ( this is taken out from html pages contains gargable and other language than english). It is given that smaller string is the larger string ( You can think smaller string as answers and larger string as the subset) I want to know how many of these pairs matched between these set of ~800 data points.
